### DISCUSSION TITLE
Consider letting pkg_manual render the examples

### DISCUSSION THREAD

**Discussion by TimTaylor** [yihui/litedown#54] [Ideas]
Would you consider an argument to `pkg_manual()` that would enable the evaluation and subsequent rendering of examples in the documentation?

**Comment by yihui** [comment]
Of course. Rendering examples was actually my main motivation when I wrote `knitr::knit_rd()` 12 years ago: https://github.com/yihui/knitr/blame/master/R/utils-rd2html.R#L45-L51

**Comment by TimTaylor** [comment]
Awesome! That's the last piece of the puzzle for my migration to litedown (although I'm sure I'll come up with a few _"nice to have"_ feature requests going forward :see_no_evil: )

**Reply by yihui** [reply]
Done. Example: https://git.yihui.org/litedown/manual.html I feel it's nice enough but feel free to suggest other "nice to have" features.
